addons/project/static/src/views/project_task_list/project_task_list_renderer.js

```diff
@@ -0,0 +1,19 @@
+/** @odoo-module */
+
+import { listView } from "@web/views/list/list_view";
+import { ListRenderer } from '@web/views/list/list_renderer';
+
+export class ProjectTaskListRenderer extends ListRenderer {
+    toggleRecordSelection(record) {
+        super.toggleRecordSelection(record);
+        if (this.props.list.selection) {
+            const project_id = JSON.stringify(this.props.list.selection[0].data.project_id);
+            const value = this.props.list.selection.every(task => JSON.stringify(task.data.project_id) === project_id) ? 'auto' : 'none';
+            document.querySelectorAll('td[name="stage_id"]').forEach(tag => {
+                tag.style.setProperty('pointer-events', value);
+            })
```

Comment:
I don't really like to do that, I will check if we cannot do the point in another method. 🤔